web-console/developer/pipelines/rbac-pipelines.md

```diff
@@ -0,0 +1,51 @@
+# Pipeline experiences when user has view only access of a project#
+
+If the user is assigned to a project with `view` access, then user will be restricted from the following actions of the project:
+ - add
+ - edit and
```

Comment:
don't need and on this line

Reply:
@serenamarie125 ,
Thanks
I will update the document and submit back. I will also recheck the pipeline.md and img file.

Reply:
@serenamarie125. I update the files. Could you please review and approve?